lib/src/widgets/reaction_popup.dart

```diff
@@ -155,10 +155,14 @@ class ReactionPopupState extends State<ReactionPopup>
   Widget get _reactionPopupRow => EmojiRow(
         onEmojiTap: (emoji) {
           widget.onTap();
-          if (currentUser != null) {
+          if (currentUser != null && _message != null) {
+            reactionPopupConfig?.onUserReacted?.call(
+              currentUser!,
```

Comment:
is this current user means message author ?

Reply:
It means a user who sends a message currently.

Reply:
Resolved

Reply:
please update the readme too

Reply:
@ujas-m-simformsolutions Current user always has control of reacting to the message. So I think there is not any requirement for the reacted user. Moreover, we are giving `Message` instance in callback so developer can find id of user who sent a message.